inception-workload/src/main/java/de/tudarmstadt/ukp/inception/workload/settings/WorkloadPanel.java

```diff
@@ -27,63 +27,55 @@
 import org.apache.wicket.markup.html.panel.Panel;
 import org.apache.wicket.model.IModel;
 import org.apache.wicket.model.Model;
-import org.apache.wicket.model.StringResourceModel;
 import org.apache.wicket.spring.injection.annot.SpringBean;
+import org.springframework.beans.factory.annotation.Autowired;
 
 import de.agilecoders.wicket.core.markup.html.bootstrap.form.BootstrapRadioChoice;
 import de.tudarmstadt.ukp.clarin.webanno.model.Project;
 import de.tudarmstadt.ukp.clarin.webanno.support.lambda.LambdaAjaxButton;
-import de.tudarmstadt.ukp.inception.workload.dynamic.manager.WorkflowProperties;
-import de.tudarmstadt.ukp.inception.workload.dynamic.manager.WorkloadProperties;
+import de.tudarmstadt.ukp.inception.workload.model.WorkloadManagementService;
+import de.tudarmstadt.ukp.inception.workload.registry.WorkloadRegistry;
 
 //Custom panel inside the page
-public class WorkflowAndMonitoringPanel extends Panel
+public class WorkloadPanel extends Panel
 {
     private static final long serialVersionUID = -6220828178550562376L;
 
-    //SpringBeans
-    private @SpringBean WorkloadProperties workloadProperties;
-    private @SpringBean WorkflowProperties workflowProperties;
-
-    private final List<String> workflow;
-    private final BootstrapRadioChoice<String> workflowChoices;
     private final List<String> monitoring;
     private final BootstrapRadioChoice<String> monitoringChoices;
+    private final Project project;
+
+    private @SpringBean WorkloadManagementService workloadManagementService;
+    private @SpringBean WorkloadRegistry workloadRegistry;
 
-    public WorkflowAndMonitoringPanel(String aID, IModel<Project> aProject) {
+    @Autowired
+    public WorkloadPanel(String aID, IModel<Project> aProject)
```

Comment:
I would expect there to be a selection of the workload type and depending on which workload is selected, a traits panel which shows the editor for the traits of the selected workload type. 

I would expect the workload type selection widget uses list of available workload managers from the workload manager registry as its choice model - similar to how it is done in the WebAnno FeatureDetailForm lines 133-183 there the `featureType` dropdown is constructed.

I would also expect that there is a `WorkoadManager` entity bean used as a model here somewhere. I.e. one in which the selected workload type is saved and where also the traits are saved by the nested traits panel.